urdfenvs/albert_reacher/resources/albert_robot.py

```diff
@@ -13,9 +13,18 @@ def __init__(self):
         self._wheel_distance = 0.494
 
     def set_joint_indices(self):
```

Comment:
After your changes, the name of this function needs to be modified to something like `set_joint_names`